subiquity/server/server.py

```diff
@@ -626,13 +656,9 @@ def use_passwd(passwd):
             use_passwd(rand_user_password())
             return
 
-        (users, _groups) = ug_util.normalize_users_groups(
-            self.cloud.cfg, self.cloud.distro)
-        (username, _user_config) = ug_util.extract_default(users)
+        username = self.installer_user_name
 
-        self.installer_user_name = username
-
-        if username is None:
+        if self.installer_user_name is None:
```

Comment:
nitpick level extreme but why change this to refer to self.installer_user_name when all the other references are still to username?